assests/styleSheets/indexStyle.css

```diff
@@ -0,0 +1,2 @@
+@import "./home.css";
+@import "./general.css";
```

Comment:
i think you need to add reset CSS here.

Reply:
such a stupid mistake I did :/
thankssss ^^